src/main/kotlin/pt/up/fe/ni/website/backend/util/Serialize.kt

```diff
@@ -0,0 +1,15 @@
+package pt.up.fe.ni.website.backend.util
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.exc.InvalidDefinitionException
+
+fun isSerializable(value: Any): Boolean {
+    return try {
+        val objectMapper = ObjectMapper()
+        objectMapper.writeValueAsString(value)
+        true
+    } catch (err: InvalidDefinitionException) {
+        System.err.println(err)
+        false
+    }
+}
```

Comment:
Do you think that'll be used outside ErrorController? It doesn't actually check if the value is serializable, it's basically used as a safe guard. I also had this problem in #88, what do you think of my approach?

![image](https://user-images.githubusercontent.com/55768934/218291819-5a883508-c25e-4d12-bf43-682673d89630.png)

Reply:
That seems like a good aproach, I will remove this serialize file